android-components/components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -1016,6 +1016,14 @@ sealed class EngineAction : BrowserAction() {
         override val tabId: String,
     ) : EngineAction(), ActionWithTab
 
+    /**
+     * Indicates the given [tabId] was unable to generate a requested save to PDF page.
+     */
+    data class SaveToPdfErrorAction(
```

Comment:
Could we rename this? Actions are usually some form of directive, past or present, (e.g. "toggle desktop mode", "exit fullscreen mode"), but here we are saying "save to pdf error" as a statement. If we use `SaveToPdfException`, it works as a past version of a directive "a save to pdf threw an exception". If we stick with `Error` it should really be `Errored` which is an odd past tense word to use, hence why `Exception` might be more fitting.

If you have other ideas too, that could work as well!

```suggestion
    data class SaveToPdfExceptionAction(
```